dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -8144,6 +8202,13 @@ definitions:
         - "KUBERNETES"
         - "HELM"
     description: "This describes one workflow version associated with a workflow."
+  WorkflowVersionPathInfo:
```

Comment:
It doesn't look like the annotations you added had any affect on the generated swagger. Did you rebuild after changing them?

Reply:
It shows up in the Models section of the swagger page

e.g:
WorkflowVersionPathInfo 
description: | Object that contains the Dockstore path to the workflow and the version tag name.
-- | --
fullWorkflowPath | stringDockstore path to workflow.
tagName | stringName of workflow version tag

Reply:
But it's not here, and this is where the Swagger UI gets the info from. Is it possible you did not commit the file? That would explain why it would show up for you locally.

Reply:
fixed